CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Version 1.12.1 (Under active development)
 
+### AppCenter
+
+#### Android
+**[Feature]** AppCenter SDK now supports a 256 character User ID string that applies to crashes, error, and push logs. 
```

Comment:
>a 256 character User ID string

This could be misleading. When I read it, I read it as the user string must be 256 characters. We should describe it so that it is clear 256 characters is maximum